src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -225,8 +220,26 @@ public function has($id)
      */
     public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE)
     {
+        $serviceNotFound = function ($id) use ($invalidBehavior) {
```

Comment:
Would `static $serviceNotFound = function ($id, $invalidBehavior) {}` be better? Otherwise extract levenshtein into `getAlternativeServiceIds()` and inline from there?